gradle-jdks/src/main/java/com/palantir/gradle/jdks/BaselineJavaJdksPlugin.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.jdks;
+
+import com.palantir.baseline.plugins.javaversions.BaselineJavaVersions;
+import com.palantir.baseline.plugins.javaversions.BaselineJavaVersionsExtension;
+import java.util.Optional;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.file.Directory;
+import org.gradle.api.provider.Provider;
+import org.gradle.jvm.toolchain.JavaInstallationMetadata;
+import org.gradle.jvm.toolchain.JavaLanguageVersion;
+
+public final class BaselineJavaJdksPlugin implements Plugin<Project> {
```

Comment:
Moved the current implementation from [`JdksPlugin`](https://github.com/palantir/gradle-jdks/pull/360/files#diff-9ca2e698760ae7c328bed1a9e7136fbe457083f7775fb90a0b870ffa7c532afaL43). This is just a copy-paste.